CHANGELOG.md

```diff
@@ -0,0 +1,71 @@
+# Release notes
+
+## 0.8.0
```

Comment:
We need to add some documentation for 0.9.0, right?

Reply:
Yes, I'm going to add the release notes in this PR.